### DISCUSSION TITLE
look up compatibility table for en_core_web_* vs spacy

### DISCUSSION THREAD

**Discussion by DSLituiev** [explosion/spaCy#13303] [Help: Installation]
I am trying to install "en_core_web_md==3.4.0" and running into an issue:
✘ No compatible package found for 'en_core_web_md==3.4.0' (spaCy v3.2.1)

I poked around with various spacy versions but not able to figure out which one this should be.
Is there a lookup table somewhere?

**Comment by DSLituiev** [comment]
One of the problems is because package manager is not able to locate `en_core_web_md==3.4.0`.
I had to run `pip install https://github.com/explosion/spacy-models/releases/download/en_core_web_md-3.4.0/en_core_web_md-3.4.0-py3-none-any.whl` for it to resolve the dependency

**Comment by svlandeg** [comment]
Hi! Usually when you run `spacy download en_core_web_md`, then the [download command](https://spacy.io/api/cli#download) should find you the right version of the pretrained model to use with your specific spaCy version. Was this not working for you correctly?

You can also find an overview of all our pretrained models for different versions [here](https://github.com/explosion/spacy-models/tree/master/meta) - it shows for each model which version of spaCy it's compatible with.
